### ISSUE TITLE
All contents "clickable" in screen reader

### ISSUE THREAD

**Issue by callumacrae** [nolimits4web/Swiper#1355]
It's kind of annoying to use Swiper using a screen reader, because it thinks everything is "clickable". This probably means you're adding an event listener to every element instead of adding it to a parent element and letting it propagate—is that the case?

**Comment by nolimits4web** [comment]
Did you try to enable `a11y`?

**Comment by callumacrae** [comment]
Nope, wasn't aware that existed. Why is Swiper inaccessible by default?

**Comment by nolimits4web** [comment]
The same, a lot developers don't event know what is accessibility API, what it for and how to use it. So it is optional feature

**Comment by callumacrae** [comment]
Oh, so it's not just a case of setting `a11y: true`? How do I use it?

**Comment by nolimits4web** [comment]
Should be enough now, fixed text labels and added a11y for pagination

**Comment by callumacrae** [comment]
:clap:
